modules/porous_flow/src/materials/PorousFlowCapillaryPressureLinear.C

```diff
@@ -0,0 +1,37 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#include "PorousFlowCapillaryPressureLinear.h"
+
+template<>
+InputParameters validParams<PorousFlowCapillaryPressureLinear>()
+{
+  InputParameters params = validParams<PorousFlowCapillaryPressureBase>();
+  params.addRequiredParam<Real>("pc_max",  "Maximum capillary pressure (Pa)");
+  params.addClassDescription("This Material provides a linear capillary pressure");
+  return params;
+}
+
+PorousFlowCapillaryPressureLinear::PorousFlowCapillaryPressureLinear(const InputParameters & parameters) :
+  PorousFlowCapillaryPressureBase(parameters),
+  _pc_max(getParam<Real>("pc_max"))
+{
+}
+
+void
+PorousFlowCapillaryPressureLinear::computeQpProperties()
+{
+  /// Capillary pressure and derivatives wrt phase saturation at the nodes
```

Comment:
Better put in an issue. I'm so busy I may not get to it right away. You
could if you want though :)
On Mon, May 9, 2016 at 5:52 PM Andy Wilkins <notifications@github.com>
wrote:

> In modules/porous_flow/src/materials/PorousFlowCapillaryPressureLinear.C
> <https://github.com/idaholab/moose/pull/6889#discussion_r62592977>:
>
> > +  InputParameters params = validParams<PorousFlowCapillaryPressureBase>();
> > +  params.addRequiredParam<Real>("pc_max",  "Maximum capillary pressure (Pa)");
> > +  params.addClassDescription("This Material provides a linear capillary pressure");
> > +  return params;
> > +}
> > +
> > +PorousFlowCapillaryPressureLinear::PorousFlowCapillaryPressureLinear(const InputParameters & parameters) :
> > +  PorousFlowCapillaryPressureBase(parameters),
> > +  _pc_max(getParam<Real>("pc_max"))
> > +{
> > +}
> > +
> > +void
> > +PorousFlowCapillaryPressureLinear::computeQpProperties()
> > +{
> > +  /// Capillary pressure and derivatives wrt phase saturation at the nodes
>
> So shall I submit an Issue which you will work on?
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/6889/files/4f2d0ae39e837aeec61169a353951bd97d029281#r62592977>
>